core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/TestOperators.scala

```diff
@@ -128,7 +128,7 @@ object TestOperators {
 
   def pythonOpDesc(): PythonUDFOpDescV2 = {
     val udf = new PythonUDFOpDescV2()
-    udf.workers = 1
+    udf.workers = None
```

Comment:
keep this test to use worker 1? I don't want this test to be dependent on the resource allocator.